tests/test_elk_reporter.py

```diff
@@ -293,7 +323,7 @@ def test_should_pass():
 
     # fnmatch_lines does an assertion internally
     result.stdout.fnmatch_lines(["*::test_should_pass PASSED*"])
-    # make sure that that we get a '0' exit code for the testsuite
+    # make sure that we get a '0' exit code for the testsuite
```

Comment:
this change is outside the intended scope of the current modifications. however, it's acceptable to include it in this iteration if necessary.

Reply:
I know, but it was bugging me :)